### DISCUSSION TITLE
英雄联盟手游

### DISCUSSION THREAD

**Discussion by Tang-Li-1** [share121/inter-knot#1550] [其他]
![#m baidu com_www baidu com](https://github.com/user-attachments/assets/fc170f7e-8682-485f-8f17-77445e530ae1)
**5v5的MOBA竞技游戏**
《英雄联盟手游》是一款5v5的MOBA竞技游戏，由美国拳头游戏开发，‌腾讯游戏K6合作部代理运营。游戏的核心玩法包括排位赛、匹配对战、训练基地和自定义模式，旨在提供多样化的游戏体验，满足不同玩家的需求。游戏中的英雄分为六大类：‌坦克、‌法师、‌刺客、‌战士、‌射手和‌辅助，每个英雄都有独特的技能和定位，使得游戏策略和战术选择更加丰富。

游戏中的主要目标是通过摧毁敌方基地来赢得对战，玩家可以选择不同的路线进行攻击，包括单人路、双人路和中路，同时还可以进入野区狩猎史诗级野怪，获得强大的增益效果。此外，游戏还提供了多种召唤师技能，如幽灵疾步、治疗术、虚弱和闪现等，这些技能在关键时刻能为玩家提供额外的生存和攻击能力。

《英雄联盟手游》在还原端游体验的基础上，针对移动端进行了适配性调整，包括操作方式的优化和游戏节奏的加快，确保玩家能在手机上享受到与电脑版相似的快感和刺激。此外，游戏还引入了星光公测等限时活动，增加了游戏的趣味性和互动性。‌

**Comment by lllliufeng** [comment]
谁问你了
